lib/sql-filter.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/* global JuttleAdapterAPI */
+let FilterJSCompiler = JuttleAdapterAPI.compiler.ASTVisitor;
+let errors = JuttleAdapterAPI.errors;
+
+const BINARY_OPS_TO_SQL_OPS = {
+    '==':  '=',
+    '!=':  '<>',
+    '=~':  'LIKE',
+    '!~':  'NOT LIKE'
+};
+
+class FilterSQLCompiler extends FilterJSCompiler {
+
+    constructor(options) {
+        super(options);
+        this.baseQuery = options.baseQuery;
+    }
+
+    compile(node) {
+        return this.visit(node, this.baseQuery);
+    }
+
+    visitExpressionFilterTerm(node, sql_query) {
+        return this.visit(node.expression, sql_query);
+    }
+
+    visitBinaryExpression(node, sql_query) {
+        let self = this;
+        let SQL_OP = BINARY_OPS_TO_SQL_OPS[node.operator] || node.operator;
+
+        if (SQL_OP === 'OR' || SQL_OP === 'AND') {
+            let whereSpecfic = SQL_OP === 'OR' ? 'orWhere' : 'where';
```

Comment:
Can you put `AND: 'where'` and `OR: 'orWhere'` in `BINARY_OPS_TO_SQL_OPS` and drop this line?